llvm/test/CodeGen/X86/GlobalISel/legalize-bswap.mir

```diff
@@ -4,35 +4,6 @@
 
 # test bswap for s16, s17, s32, and s64
 
-...
----
-name:            test_bswap17
```

Comment:
change to bswap48 instead of removing?

Reply:
Didn't see your PR description. NVM

Reply:
As I wrote in my description, I tried. I hit some other legalizer issue.

Reply:
cc @e-kud

Reply:
Yeah, `X86` implementation was copied from `AArch64` that also fails on the following mir with the same assert
```
 %0:_(s48) = IMPLICIT_DEF
 %1:_(s16), %2:_(s16), %3:_(s16) = G_UNMERGE_VALUES %0:_(s48)
 %4:_(s32) = G_ANYEXT %1(s16)
 $w0 = COPY %4(s32)
 RET_ReallyLR implicit $w0
```